rules/javascript/express/open_redirect.yml

```diff
@@ -2,8 +2,12 @@ imports:
   - javascript_shared_common_user_input
 patterns:
   - pattern: |
-      res.redirect($<USER_INPUT>$<...>)
+      $<RES>.redirect($<USER_INPUT>$<...>)
```

Comment:
Not sure how we'd feel about making this totally agnostic e.g. `$<_>`? /cc @cfabianski

Reply:
Maybe a regex with ˋ\Ares(ponse)?ˋ ?

Reply:
Nice - I like this